src/Model/ExitManager.php

```diff
@@ -67,8 +89,8 @@ public function exitsFiltered($filter)
             };
             $query = "SELECT exit.name, exit.image, exit.department, exit.height, exit.id 
             from `exit`
-            left join `exit_Has_Type_Jump` on `id_exit`=exit.id
-            left join `type_Jump` on `id_type_jump`=type_jump.id
+            left join `exit_has_type_jump` on `id_exit`=exit.id
```

Comment:
If you create a constant for your join table (TABLE_HAS_TYPE_JUMP), you should use to join your table